apps/api/src/test-data/buildTermTestData.ts

```diff
@@ -163,6 +163,7 @@ export const buildTermsForVocabularyList = () =>
     (
         [
             [iAmSingingEvents, '511'],
+            // should this be '521' ?
```

Comment:
Remove comment in favour of explaining that: '*12" means 1 for positive, 2 for second person. They are all singular.

Reply:
done